src/dmd/dsymbolsem.d

```diff
@@ -1439,7 +1439,9 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
                 {
                     uint errors = global.errors;
                     dsym.inuse++;
-                    if (ei)
+                           // Bug 20549. Don't try this on modules or packages, syntaxCopy
+                           // could crash (inf. recursion) on a mod/pkg referencing itself
```

Comment:
please align the comments with the above.

Reply:
The intent was to comment the extra condition.

Reply:
Sorry I should have said please unindednt the comment.

Reply:
Ok